pygmt/datasets/load_earth_dataset.py

```diff
@@ -0,0 +1,204 @@
+"""
```

Comment:
Also need to rename this file to `load_remote_dataset.py`.

Reply:
Done